linux_docker_resources/Dockerfile

```diff
@@ -69,9 +69,9 @@ RUN if test ${COMPILE_WITH_CLANG} = true; then apt-get update && apt-get install
 RUN apt-get update && apt-get install --no-install-recommends -y gcovr
 
 # Install the OpenSplice binary from the OSRF repositories.
-RUN if test \( ${PLATFORM} = arm -o ${PLATFORM} = x86 \) ; then apt-get update && apt-get install --no-install-recommends -y libopensplice69=6.9.190403+osrf1-1~$UBUNTU_DISTRO; fi
+RUN if test \( ${PLATFORM} = aarch64 -o ${PLATFORM} = armhf -o ${PLATFORM} = x86 \) ; then apt-get update && apt-get install --no-install-recommends -y libopensplice69=6.9.190403+osrf1-1~$UBUNTU_DISTRO; fi
 # Update default domain id.
-RUN if test \( ${PLATFORM} = arm -o ${PLATFORM} = x86 \) ; then sed -i "s/<Id>0<\/Id>/<Id>108<\/Id>/" /usr/etc/opensplice/config/ospl.xml; fi
+RUN if test \( ${PLATFORM} = aarch64 -o ${PLATFORM} = x86 \) ; then sed -i "s/<Id>0<\/Id>/<Id>108<\/Id>/" /usr/etc/opensplice/config/ospl.xml; fi
```

Comment:
IIUC the change above this one installs opensplice if the platform is `aarch64`, `armhf`, or `x86`. Should this line be modified to change `ospl.xml` when the platform is `armhf` too?

Reply:
I don't see why not. What platform do we build besides `aarch64`, `armhf` and `x86`?

Reply:
I can't think of any. I do see job names on https://ci.ros2.org/ of `ci_linux-aarch64` and `ci_linux-centos`. Are our scripts treating CentOS as a different platform?

Reply:
CentOS has never used this Dockerfile. Even if it did, I think `PLATFORM` would be set to `x86` in this case.

Reply:
I can't find any mention of any PLATFORM value other than `aarch64`, `armhf` or `x86`. I think this is just a remnant of when we didn't install OpenSplice on one or more ARM platforms. I'll remove it, and trigger CI for all platforms.